spec/factories/user.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    name { "John" }
+    email { "john#{rand(100)}@email.com" }
```

Comment:
Is the idea here to get unique emails for different users? This might fail if you try to create many users in a test, and `rand` returns the same number twice. You can use factorybot's [sequences](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences) to make this deterministic.

Separately, it's usually a good idea to avoid randomness in the factory because it can make tests non deterministic. If a specific test needs random data, it's better to pass that in from the test and keep the factory simple. It makes debugging simpler.

Reply:
Got it.  I will use sequences instead of rand.